torchvision/datasets/fgvc_aircraft.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+import os
+from typing import Any, Callable, List, Optional, Tuple
+
+import PIL.Image
+
+from .utils import download_and_extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+annotation_level_to_file = {
+    "variant": "variants.txt",
+    "family": "families.txt",
+    "manufacturer": "manufacturers.txt",
+}
```

Comment:
Given that we only talk about a dictionary with three items that is used in a single location, maybe we can inline it?

```python
file_name = {
    "variant": "variants.txt",
    "family": "families.txt",
    "manufacturer": "manufacturers.txt",
}[self._annotation_level]
annotation_file = os.path.join(input_path, "data", file_name)
```